spec/models/user_spec.rb

```diff
@@ -2,4 +2,11 @@
 
 RSpec.describe User, type: :model do
 
+  context 'email' do
+    it "must be unique" do
+      user1 = FactoryBot.create(:user, email: 'hello@email.com' )
+      user2 = FactoryBot.build(:user, email: 'hello@email.com' )
+      expect(user2).to be_invalid
```

Comment:
Yep this works. You probably also want to ensure there is an email too, right?
A stylistic suggestion that I quite like for validations is to make use of the rspec subject and the `is_expected` syntax which works nicely with the `it`:

```
describe 'validations' do
  context 'when email is missing' do
    subject(:user) { FactoryBot.build(:user) }
    before do
      subject.email = nil
    end
    it { is_expected.to be_invalid }
  end
  
  context 'when email is already taken' do
    let!(:user1) { FactoryBot.create(:user, email: 'hello@email.com' ) }
    subject(:user2) { FactoryBot.build(:user, email: 'hello@email.com' ) }
    it { is_expected.to be_invalid }
  end
end
```